proto/vtctldata.proto

```diff
@@ -208,7 +208,8 @@ message Shard {
 
 message WorkflowOptions {
   string tenant_id = 1;
-  string source_keyspace_alias = 2;
+  // Previously used for source_keyspace_alias which was removed before being formally released.
+  reserved 2;
```

Comment:
Do we need to mark this reserved or can we just delete it and move `strip_sharded_auto_increment` to `2` since that PR is also recent and we haven't got a release or a production cluster with this proto in use.

Reply:
I think we can change the proto since neither of those fields were in a release.

Reply:
agree with mattlord. we can rearrange the fields.